e2e/Makefile

```diff
@@ -34,11 +35,16 @@ deploy-operator:
 		cp -r "../hack" "../operator" "../server" "$$f" && \
 		cd "$$f/operator" && \
 		( \
-			cd config/manager && \
+			pushd config/manager && \
 			( \
 				toolchain_host=$$($(KUBECLI) get namespaces -o name | grep toolchain-host | cut -d'/' -f2 | head -n 1); \
 				$(KUSTOMIZE) edit add configmap operator-config --behavior=replace \
-					--from-literal=kubesaw.namespace=$$(( [[ -n "$$toolchain_host" ]] && echo "$$toolchain_host" ) || echo "workspaces-system" ) \
+					--from-literal=kubesaw.namespace=$$(( [[ -n "$$toolchain_host" ]] && echo "$$toolchain_host" ) || echo "$(NAMESPACE)" ) \
+			) && \
+			popd && \
+			pushd config/default && \
+			( \
+				$(KUSTOMIZE) edit set namespace $(NAMESPACE) \
```

Comment:
```suggestion
			( cd config/default && $(KUSTOMIZE) edit set namespace $(NAMESPACE) \
```

Reply:
Fixed in 67cadbd.

Couldn't take this change as-is, since there's another usage of `pushd` above this patch.